mods/engine/bs_drops/init.lua

```diff
@@ -46,3 +46,16 @@ core.register_on_punchplayer(function(player, hitter, time_from_last_punch, tool
 end)
 
 minetest.register_on_leaveplayer(dropondie.drop_all)
+
+--[[
```

Comment:
this multiline can be removed, it's just the original code I pulled to rework for this bugfix.